src/components/common/Header/Header.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import glossarioLogo from '../../../../assets/images/glossario-logo.svg'
+import { Search } from '../index'
+import { Link } from 'react-router-dom'
+import terms from '../../../lib/data';
+
+import './Header.css'
+
+const DayPhrase = ({ entry }) => {
+  return (
+    <span className={'header__day-phrase'}>
+      Você sabe o que é&nbsp;
+      <Link
+        className='emphasis pointer-hover light-accent lighter-hover'
+        to={`/${entry}`}
+      >
+        {entry}
+      </Link>
+      ?
+    </span>
+  );
+};
+
+DayPhrase.propTypes = {
+  entry: PropTypes.string
+}
+
+export default function Header(props) {
```

Comment:
Pode colocar numa const e exportar depois? Só pra ser coerente com as outras definicoes. Algo como 
```suggestion
const Header = (props) => {}
export default Header
```

Reply:
Ops, foi